content/webapp/components/ExhibitionGuideLinks/ExhibitionGuideLinks.tsx

```diff
@@ -112,4 +97,61 @@ const ExhibitionGuideLinks: FunctionComponent<Props> = ({
   );
 };
 
-export default ExhibitionGuideLinks;
+type ResourceProps = {
+  textPathname: string | undefined; // TODO make these more specifid=c?
```

Comment:
Is still still todo? 

And do we have a convention/preference on

`textPathname?: string`
vs
`textPathname: string | undefined`?